packages/sources/css/src/components/toggle/src/index.css

```diff
@@ -0,0 +1,74 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+/* The default switch - the box around the slider */
+.vtmn-toggle_switch {
+  position: relative;
+  display: inline-block;
+  width: rem(56px);
+  height: rem(32px);
+}
+
+/* Hide default HTML checkbox */
+.vtmn-toggle {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The default slider */
+.vtmn-toggle_slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--vtmn-color_grey-light-1);
+  transition: 0.4s;
```

Comment:
We can transition only what's necessary:
`transition: background-color 0.4s;` here ...

Reply:
Ok thanks :)